### ISSUE TITLE
Topic Idea: Creating an agency-ready block theme

### ISSUE THREAD

**Issue by ghost** [WordPress/Learn#1272]
# Details
- Topic title: Creating an agency-ready block theme
- Topic description: I would like to create a course that combines the existing courses and adds to it, by discussing how you can include multiple (dynamic) blocks in your theme. There's no course on dynamic blocks yet, and also none on how to combine multiple blocks in a plugin nor a theme. We would also extend the `@wordpress/scripts` package to have multiple blocks registered in a theme.
- Objective: Creating an agency-ready block theme
- Content type: Advanced Developers
- WordPress version: Latest
- Will you be creating this content?: Yes

# Related Resources
Links to related content on Learn, HelpHub, DevHub, GitHub Gutenberg Issues, DevNotes, etc.
- We would go for a barebones version of [this setup](https://github.com/thomasdevisser/stuurlui)

# Next steps for SMEs
Please follow the team handbook "[Vetting topic ideas](https://make.wordpress.org/training/handbook/training-team-how-to-guides/vetting-topic-ideas/)" to vet this topic.

**Comment by annezazu** [comment]
# Vetting Checklist for SMEs
- Priority: medium.
- Feedback regarding the topic description: I'd approach it more from the point of view of curation and adding additional functionality to a block theme rather than "agency ready" per say. 
- Feedback regarding the learning objectives: N/A
- Any other feedback: I'd recommend having it as part 3 of this series https://learn.wordpress.org/course/a-developers-guide-to-block-themes-part-1/ and would show how TT4 makes use of custom block variations, pulling patterns from theme.json, alongside showing dynamic blocks built in. It might also be wise to touch on curation pathways as well and showing a few as part of the larger part 3 effort: https://developer.wordpress.org/block-editor/how-to-guides/curating-the-editor-experience/ cc @jonathanbossenger for thoughts there.

**Comment by jonathanbossenger** [comment]
```
 I'd approach it more from the point of view of curation and adding additional functionality to a block theme rather than "agency ready" per say.
```

While I understand where the original idea came from, I agree with this idea. By focusing on the topics of curation and adding functionality.

I actually created a similar course idea issue [here](https://github.com/WordPress/Learn/issues/1263) which includes a lot of the same ideas/discussions. In August I updated it with an overview of the [suggested topics](https://github.com/WordPress/Learn/issues/1263#issuecomment-1697598422) for such a course.

Last but not least, I would remind folks that the [Theme Developer Learning Pathway](https://github.com/WordPress/Learn/issues/2045) also contains most, if not all, of these topics. 

With our (the Training Team) focus switching to the learning pathways, it might be more beneficial to review that list, ensure that anything in this issue, or the other one I mentioned, is included in the learning pathway, and then focus on creating that content. 

@cynthianorman as you have expressed interest in creating content for the theme developer learning pathway, these discussions might be of interest to you.
